src/processor/Package.ts

```diff
@@ -388,4 +389,27 @@ export class Package {
       );
     }
   }
+
+  private combineContainsRules(): void {
+    [...this.profiles, ...this.extensions].forEach(sd => {
+      const rulesToRemove: number[] = [];
+      sd.rules.forEach((rule, i) => {
+        if (rule instanceof ExportableContainsRule && !rulesToRemove.includes(i)) {
+          sd.rules.forEach((otherRule, otherRuleIdx) => {
+            if (
+              otherRule.path === rule.path &&
+              otherRule instanceof ExportableContainsRule &&
```

Comment:
I'm not entirely sure if this matters, but I feel like I remember talking once about swapping the order of these checks, so that it checks the `instanceof` before looking at the path. I don't exactly remember the context of that conversation, so if you remember and don't think it's relevant, it's all good with me.

Reply:
Hm yeah I do remember that conversation. I looked throughout the file, and there is variability in the order of us checking `path` and doing an `instanceof` check. I guess personally I must like doing `path` first because that is what I subconsciously did here, but I think @cmoesel had been a fan of doing `instanceof` (first)[https://github.com/FHIR/GoFSH/pull/10/files/890554e1a512a91249980dfdeec14f585ef691bb#diff-bf3b12f81fc71976ec2cec060d7dc5c9ecb4a186adbfd9bbb4d858c4e38dbd98]. So if @cmoesel will let this one slide, I'm ok with it.

Reply:
Nick wins: https://jsben.ch/IpyO7

Reply:
I ran that four times and you each won twice 🤷‍♀️

Reply:
Participation medals for all 🏅 !

Reply:
I ran it twice and I lost both times. :-(

But I am using Firefox.  Is @jafeltra using Chrome?  If so, maybe the Chrome JS implementation isn't as definitive.

Reply:
I was using Chrome. So maybe!